### ISSUE TITLE
Photo for basic.md?

### ISSUE THREAD

**Issue by benjaminapetersen** [devonjones/openforge-tutorials#3]
If you have an image handy, it may be really helpful to see what the suggested set can make.

**Comment by devonjones** [comment]
Yeah, good idea.  I'll put something together for that.  Now I just need to see if I actually have any tile set where I've actually got a full basic :-P 

In many cases I've printed what I need to get a painted release out, so in some cases I have not yet actually even filled out a full basic set.  That being said, that's fairly easily resolved.
